### PR TITLE
Treasure567

### PR DESCRIPTION
**Pull request by treasure567**
User can login and Register

### CODE REVIEW

**Review by ahmadinjo — CHANGES_REQUESTED** [review]

File: src/database/queries.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255) NOT NULL UNIQUE,
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
+)`;
+
+const createTableProperties = `
+CREATE TABLE IF NOT EXISTS properties (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    title VARCHAR(255) NOT NULL,
```
[the remark this answers is not in the document]
Comment (treasure567) [reply src/database/queries.js:31]: I was following specifications given from the capstone

File: src/database/queries.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    phone VARCHAR(20) NOT NULL, 
+    address VARCHAR(255), 
+    is_admin BOOLEAN DEFAULT FALSE, 
+    created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
+)`;
+const createTableCategories = `
+CREATE TABLE IF NOT EXISTS categories (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    name VARCHAR(255) NOT NULL UNIQUE,
```
[the remark this answers is not in the document]
Comment (treasure567) [reply src/database/queries.js:23]: Ok cool

File: src/database/queries.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+)`;
+
+const createTableProperties = `
+CREATE TABLE IF NOT EXISTS properties (
+    id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+    title VARCHAR(255) NOT NULL,
+    user_id INT NOT NULL,
+    category_id INT NOT NULL,
+    price INT NOT NULL,
+    description VARCHAR(255) DEFAULT NULL,
```
[the remark this answers is not in the document]
Comment (treasure567) [reply src/database/queries.js:35]: Noted. Will update

File: src/database/queries.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    image VARCHAR(255) DEFAULT NULL,
```
[the remark this answers is not in the document]
Comment (treasure567) [reply src/database/queries.js:36]: Ok boss

File: src/database/queries.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (ahmadinjo) [reply src/database/queries.js:31]: No problem then. Leave it at that

**Comment by treasure567** [comment]
@kelue Review and approve

**Review by kelue — APPROVED** [review]
Looks good

**Review by ahmadinjo — APPROVED** [review]
Good work boss. I have effect some changes in my pull request too, you can check them out

**Pull request merged**